Library/Formula/icetouch.rb

```diff
@@ -0,0 +1,22 @@
+class Icetouch < Formula
+  homepage "https://zeroc.com"
+  url "https://github.com/zeroc-ice/icetouch.git",
+    :tag => "v3.6.0", :revision => "f443edded03fac7f4e5c353ebc87efaf4bfa5229"
+
+  depends_on "mcpp"
+
+  def install
+    # Unset ICE_HOME as it interferes with the build
+    ENV.delete("ICE_HOME")
+    ENV.delete("USE_BIN_DIST")
+    ENV.delete("CPPFLAGS")
```

Comment:
I did some research, as this formula we've had around for a while. The issue was that ENV.02 was being overwritten by brew's CPPFLAGS. This seems to have been fixed though. I'll remove it from my formula.